tests/cloudformation/checks/resource/aws/example_ECRPolicy/ECRPolicy-PASSED-2.yaml

```diff
@@ -0,0 +1,28 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources: 
+  MyRepository: 
+    Type: AWS::ECR::Repository
+    Properties: 
+      RepositoryName: "test-repository"
+      RepositoryPolicyText: 
+        Version: "2012-10-17"
+        Statement: 
+          - 
+            Sid: AllowPushPull
+            Effect: Allow
+            Principal: "*"
+            #Principal: 
+            #   AWS: 
+            #     - "arn:aws:iam::123456789012:user/Bob"
+            #     - "arn:aws:iam::123456789012:user/Alice"
```

Comment:
just remove the comment, not needed anymore 🙂

Reply:
Done